### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by sebko**
[Link to rendered markdown](https://github.com/mirainc/custom-menu-samples/blob/9b456b1ffdd26b45bb89c5ca5b1b899aa6e06d3a/gotts-menu/README.md)

### CODE REVIEW

File: gotts-menu/README.md [hunk 1]
```diff
@@ -27,17 +31,107 @@ See [here](https://raydiant.notion.site/How-do-I-develop-an-app-487d7064eeec402f
...
+
+```
+/v1/groups?tags=<tag>&menus=<menuId>depth=<depth>
+```
+
+### Modifiers
+
+Toast modifiers aren't fully supported yet. Modifier-like behaviour can be achieved through tags + items.
+In the example below, there's a total of three different modifiers
+![modifiers](modifiers.png)
```
Comment (jschr) [inline gotts-menu/README.md:116]: I'm not sure if this is entirely accurate. We do support Toast modifiers (1-level deep) but not modifier groups.

File: gotts-menu/README.md [hunk 2]
```diff
@@ -27,17 +31,107 @@ See [here](https://raydiant.notion.site/How-do-I-develop-an-app-487d7064eeec402f
...
+
+1. Price sizing
+2. Flavours
+3. Additional Flavours
```
Comment (jschr) [inline gotts-menu/README.md:120]: Minor but I'm thinking we should probably use "american" spelling for our documentation.

Comment (sebko) [reply gotts-menu/README.md:120]: Yeah, we'll definitely want these in american-english. Good catch! I'll run the whole document through a spelling checker 🙏

File: gotts-menu/README.md (hunk 1 above)
Comment (sebko) [reply gotts-menu/README.md]: Totally missed that. Cheers! Updated

File: gotts-menu/README.md (hunk 2 above)
Comment (sebko) [reply gotts-menu/README.md]: I'll also need to make sure I change the flavour tags in Toast after this has been merged.
https://github.com/mirainc/custom-menu-samples/pull/16/commits/b5cda5b174c633bf2486229736db853048f55530

File: gotts-menu/README.md (hunk 1 above)
Comment (sebko) [reply gotts-menu/README.md]: https://github.com/mirainc/custom-menu-samples/pull/16/commits/7a651d02f73b4bb626956ec594d0b1e617b818b8

**Review by jschr — APPROVED** [review]

**Pull request merged**